src/main/java/lotto/RawExpression.java

```diff
@@ -0,0 +1,35 @@
+package lotto;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RawExpression {
+
+    private String customDelimiter;
+    private String expression;
+    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\n(.*)";
+    private static final Integer MATCH_GROUP_DELIMITER = 1;
+    private static final Integer MATCH_GROUP_EXPRESSION = 2;
```

Comment:
인스턴스 선언시, 상수 다음 인스턴스 변수가 오는게 관례이오니 참고 부탁드릴께요 😄 
```suggestion
    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\n(.*)";
    private static final Integer MATCH_GROUP_DELIMITER = 1;
    private static final Integer MATCH_GROUP_EXPRESSION = 2;
    
    private String customDelimiter;
    private String expression;

```

Reply:
네 감사합니다!